dumpling/export/dump.go

```diff
@@ -1044,6 +1044,10 @@ func getListTableTypeByConf(conf *Config) listTableType {
 }
 
 func prepareTableListToDump(tctx *tcontext.Context, conf *Config, db *sql.Conn) error {
+	if conf.Tables != nil {
```

Comment:
`conf.Tables` will never be `nil` in your function.
```suggestion
	if !conf.specifiedTables {
```